### ISSUE TITLE
Great project for a Library

### ISSUE THREAD

**Issue by marcgauthier** [weiserhei/panoglobe#5]
Do you have any plan to turn your code into a JS library?

**Comment by weiserhei** [comment]
No. It wouldnt be pleasant for me as dev and the users as Im not consistent and changing a whole lot of things while improving this app. My long-term thought was offering this as a free service to put in your own routes and share with people.

However, there is some library from google to build data globes, maybe thats working for you.
